packages/instant-apps-components/src/instant-apps-landing-page-2.html

```diff
@@ -51,6 +51,7 @@
     <instant-apps-landing-page
       id="landingPage"
       title-text="ArcGIS Instant Apps"
+      subtitle-text="Lorem ipsum dolor sit amet"
```

Comment:
Is this just for testing? We probably don't want lorem ipsum in the app

Reply:
This is just the test file